downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -425,6 +425,7 @@ enum class VcsHost(
          */
         fun toArchiveDownloadUrl(vcsInfo: VcsInfo): String? {
             val normalizedVcsInfo = vcsInfo.normalize()
+            if (normalizedVcsInfo.revision.isEmpty()) return null
```

Comment:
This logic is only used in the analyzer, while the code is located in the `downloader`.

The `fix(downloader)` prefix, IMO can easily be misleading one into thinking that this fixes something in the downloader. However, it fixes an issue in the analyzer result. Would you be fine using `fix(analyzer)` as the prefix ?

Reply:
Well, the `downloader` is a library, and as such it has no knowledge about where its code is being called from. So I believe it's not meaningful to change the scope to the name of the (currently) only caller. But I'll think about adding more detail to the commit message.

Reply:
> This logic is only used in the analyzer

Actually, that's not true, it's also used with `ort download --dry-run`.